public/app/features/admin/Users/sort.ts

```diff
@@ -0,0 +1,28 @@
+import { SortByFn } from '@grafana/ui';
+
+/**
+ * Create a function that sorts by a given key, with the following rules:
+ * - If the value is a number, sorts by number
+ * - If the value is a string, sorts by date if it can be parsed as a date
+ * - If the value is a boolean, sorts by boolean
+ * - Otherwise, sorts by string
+ */
+export const createSortFn =
```

Comment:
So apparently the table has default sorting types available and this function is largely unnecessary 🙈 Will check if any parts of it are still needed.